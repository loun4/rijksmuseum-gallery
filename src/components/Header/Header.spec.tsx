import { render } from '@testing-library/react'
import { describe, expect, it } from 'vitest'
import { Header } from './index'

describe('Header', () => {
  it('render with logo', () => {
    const { getByAltText } = render(<Header />)
    expect(getByAltText('RijksData')).toBeDefined()
  })
})
