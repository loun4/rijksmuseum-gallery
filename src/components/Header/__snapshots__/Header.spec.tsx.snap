// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Header > render 1`] = `
<DocumentFragment>
  <header
    class="Header"
  >
    <img
      alt="RijksData"
      height="32"
      src="/src/assets/logo.png"
      title="RijksData"
      width="200"
    />
    <form
      class="Search"
    >
      <div
        class="Search__inputWrapper"
      >
        <input
          aria-label="Search"
          class="Search__input"
          placeholder="Search..."
          type="text"
          value=""
        />
      </div>
      <input
        class="Search__submit"
        disabled=""
        type="submit"
        value="Submit"
      />
    </form>
  </header>
</DocumentFragment>
`;
